### PR TITLE
showing selected statistic string in y-axis label of spectral profiler

### LINKED ISSUE #256: spectral profiler change y-axis label to reflect statistic
When you change the statistic being displayed the label of the y axis on the display should change to match. Instead of generic “value” replace with “mean” or “RMS” or whatever stat is being display. “Value” is the appropriate label only for “cursor”.

### PR DESCRIPTION
**Pull request by TienHao**
closes #256

### CODE REVIEW

**Review by kswang1029 — APPROVED** [review]
Tested and it looks good. 

UPDATE: found a bug please refer to the following comment.

**Review by kswang1029 — CHANGES_REQUESTED** [review]
When we have a point region and a rectangle region with FluxDensity as statistic, switching between these two region results in the wrong unit in the point region profile (should be Jy/beam but it is Jy).

File: src/components/SpectralProfiler/SpectralProfilerComponent.tsx [hunk 1]
```diff
@@ -380,13 +380,25 @@ export class SpectralProfilerComponent extends React.Component<WidgetProps> {
...
+                let yLabelName = "Value";
+                let yLabelUnit = "";
+                let region: RegionStore;
+                if (frame.regionSet) {
+                    region = frame.regionSet.regions.find(r => r.regionId === this.widgetStore.effectiveRegionId);
+                    if (region.regionType !== CARTA.RegionType.POINT) {
+                        yLabelName = SpectralProfileWidgetStore.StatsTypeString(this.widgetStore.statsType);
+                    }
+                }
+                if (region && region.regionType === CARTA.RegionType.POINT) {
```
Comment (veggiesaurus) [inline src/components/SpectralProfiler/SpectralProfilerComponent.tsx]: couldn't this be more elegantly combined with the `if` statement above?

**Comment by veggiesaurus** [comment]
@kswang1029 are you happy with these changes?

**Comment by kswang1029** [comment]
@veggiesaurus @TienHao tested 6bb934b and it works well. 👍

**Pull request merged**